pyproject.toml

```diff
@@ -55,11 +55,12 @@ cleo = "^2.0.0"
 click = "^8.1.3"
 numpy = "^1.21.2"
 einops = "^0.6.0"
-transformers = "^4.29.0"
+transformers = { git = "https://github.com/huggingface/transformers.git", branch = "main" }
```

Comment:
reject, I knew it only works on the latest transformer. But I would not mention it in our dependency.

Reply:
just wait for a while the main branch is released. Then, we just bump the tag version.